.github/CODEOWNERS

```diff
@@ -938,6 +938,13 @@
 # ServiceLabel: %Astronomer %Mgmt
 # ServiceOwners:                                                   @banggaurav
 
+# PRLabel: %New Relic
```

Comment:
Please move this block down to ~L981 to preserve alphabetical ordering by label.

Reply:
Done